src/jdk.management/share/classes/com/sun/management/doc-files/threadDump.schema.json

```diff
@@ -78,6 +78,10 @@
                               "description": "The blocker object responsible for the thread parking."
                             }
                           },
+                          "owner": {
+                            "type": "string",
+                            "description": "The thread identifier of the owner when the parkBlocker is an AbstractOwnableSynchronizer."
+                          }
                           "required": [
                             "object"
                           ]
```

Comment:
At L122 the null in the items object should be in quotes, as in "null", and the description ending in ".. null if eliminated" needs a period. There were in the first prototype but didn't make it into the PR patch for some reason. (It came up when using the updated JSON schema, wasn't noticed previously).